spec/components/guardian/user_guardian_spec.rb

```diff
@@ -273,7 +273,7 @@
         expect(guardian.can_delete_user?(user)).to eq(true)
       end
 
-      it "is allowed when user created multiple posts in PMs to themself" do
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617143041
`themself` is archaic